articles.tex

```diff
@@ -311,7 +310,8 @@
 \asubsection{Non-Voting Members}
 \begin{itemize}
 	\item Ad Hoc Directors
-\asubsection{House Secretary}
+        \item House Secretary
+        \item Chairman
```

Comment:
I would put Chairman, then House Secretary, then Ad Hoc Directors personally, but that's not required and can be changed non-semantically later.